src/nc_message.h

```diff
@@ -141,6 +142,7 @@ typedef enum msg_parse_result {
     ACTION( REQ_REDIS_RPUSH )                                                                       \
     ACTION( REQ_REDIS_RPUSHX )                                                                      \
     ACTION( REQ_REDIS_SADD )                   /* redis requests - sets */                          \
+    ACTION( REQ_REDIS_SADDINT )                                                                       \
```

Comment:
SADDINT isn't merged into redis, even in the unstable branch. I'm not sure what this is.

Reply:
you may visit the [PR at keydb](https://github.com/EQ-Alpha/KeyDB/pull/354)